### PR TITLE
pscanrules: More http to https link tweaks

### PR DESCRIPTION
**Pull request by kingthorin**
## Overview
- CHANGELOG > Added change note.
- Messages.properties > Update resources.

## Related Issues
- zaproxy/zaproxy#8262

## Checklist
- [NA] Update help
- [x] Update changelog
- [x] Run `./gradlew spotlessApply` for code formatting
- [NA] Write tests
- [NA] Check code coverage
- [x] Sign-off commits
- [x] Squash commits
- [x] Use a descriptive title

### CODE REVIEW

File: addOns/pscanrules/CHANGELOG.md [hunk 1]
```diff
@@ -4,7 +4,11 @@ All notable changes to this add-on will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## Unreleased
-
+### Changed
+- Use of HTTP for example URLs in the descriptions for the following rules have been updated to HTTPS (Issue 8262):
```
Comment (thc202) [inline addOns/pscanrules/CHANGELOG.md:8]: nit, in some cases it's other info.

Comment (kingthorin) [reply addOns/pscanrules/CHANGELOG.md:8]: Can adjust

**Comment by kingthorin** [comment]
Tweaked

**Review by thc202 — APPROVED** [review]

**Comment by thc202** [comment]
Thank you!

**Review by psiinon — APPROVED** [review]

**Pull request merged**